### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by ganesh-sanap**
fixed typos in article.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/business-video/moveto-microsoft-365/mover-migrate-files.md [hunk 1]
```diff
@@ -27,20 +27,20 @@ description: "Learn how to Migrate Google files to Microsoft 365 for business by
...
+When you move to Microsoft 365 for business, you'll want to migrate your files from Google Drive. You can use the Mover app to move files from personal and shared Drives. For more information, see [Mover Cloud Migration](https://docs.microsoft.com/sharepointmigration/mover-plan-migration).
 
 > [!NOTE]
 > Mover will make a copy of the files and move the copies to Microsoft 365 for business. The original files will stay in Google Drives also.
 
 ## Before you start
 
-All the users should have signed in to Microsoft 365 for business and set up their OneDrive for Business. To do this, go to [office.com](https://office.com), sign in with you Microsft 365 for business credentials, and then choose OneDrive.
+All the users should have signed in to Microsoft 365 for business and set up their OneDrive for Business. To do this, go to [office.com](https://office.com), sign in with you Microsoft 365 for business credentials, and then choose OneDrive.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/moveto-microsoft-365/mover-migrate-files.md]:
```suggestion
All the users should have signed in to Microsoft 365 for business and set up their OneDrive for Business. To do this, go to [office.com](https://office.com), sign in with your Microsoft 365 for business credentials, and then choose OneDrive.
```

File: microsoft-365/business-video/moveto-microsoft-365/mover-migrate-files.md [hunk 2]
```diff
@@ -27,20 +27,20 @@ description: "Learn how to Migrate Google files to Microsoft 365 for business by
...
 
 ## Try it!
 
 ### Install Mover
 
-1. Sign into your Google Workspace admin console at [admin.google.com](https://admin.google.com).
+1. Sign in to your Google Workspace admin console at [admin.google.com](https://admin.google.com).
 
 1. Choose **Apps**, **Google Workspace Marketplace apps**, Then **Add app to Domain Install list**.
```
Comment (JohanFreelancer9) [inline microsoft-365/business-video/moveto-microsoft-365/mover-migrate-files.md]:
```suggestion
1. Choose **Apps** > **Google Workspace Marketplace apps** > **Add app to Domain Install list**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks.

**Pull request merged**